scripts/create-test-vm.mjs

```diff
@@ -0,0 +1,39 @@
+import { execSync } from 'child_process';
+import { fileURLToPath } from 'url';
+import { dirname, resolve } from 'path';
+
+// Function to run Terraform commands
+const runTerraform = (prId) => {
+  try {
+    console.log(`Starting to create the test VM with PR ID: ${prId}`);
+
+    // Resolve the current directory of this file
```

Comment:
Tip: Since this (finding out where on the filesystem we are) is a common challenge in a number of scripts, it's available as an import form the config folder:

```mjs
import  { MORIO_GIT_ROOT } from '../config/cli.mjs'

// MORIO_GIT_ROOT now holds the full path to the root folder of the git repository
```